### ISSUE TITLE
Errors building xt.min.js

### ISSUE THREAD

**Issue by bradh** [ngageoint/opensphere#617]
I'm trying to build the XT module using `yarn build:xt` however it doesn't generate the `xt.min.js` module. Instead I get:

```
$ mkdirp .build dist/opensphere; os-compile $(cat src/os/xt/gcc-args); cp src/os/xt/xt-example.html dist/opensphere
src/os/alert/alertevent.js:5: ERROR - required "goog.date.DateTime" namespace never provided
goog.require('goog.date.DateTime');
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/os/alert/alertevent.js:6: ERROR - required "goog.events.Event" namespace never provided
goog.require('goog.events.Event');
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/os/alert/alertevent.js:7: ERROR - required "goog.events.EventTarget" namespace never provided
goog.require('goog.events.EventTarget');
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^


src/os/alert/alertmanager.js:3: ERROR - required "goog.events.EventTarget" namespace never provided
goog.require('goog.events.EventTarget');
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/os/alert/alertmanager.js:4: ERROR - required "goog.log" namespace never provided
goog.require('goog.log');
^^^^^^^^^^^^^^^^^^^^^^^^

src/os/alert/alertmanager.js:5: ERROR - required "goog.structs.CircularBuffer" namespace never provided
goog.require('goog.structs.CircularBuffer');
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/os/alert/alertmanager.js:8: ERROR - required "os.array" namespace never provided
goog.require('os.array');
^^^^^^^^^^^^^^^^^^^^^^^^

src/os/xt/events.js:3: ERROR - required "goog.events.Event" namespace never provided
goog.require('goog.events.Event');
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/os/xt/events.js:4: ERROR - required "goog.events.EventTarget" namespace never provided
goog.require('goog.events.EventTarget');
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/os/xt/peer.js:4: ERROR - required "goog.Timer" namespace never provided
goog.require('goog.Timer');
^^^^^^^^^^^^^^^^^^^^^^^^^^

src/os/xt/peer.js:5: ERROR - required "goog.array" namespace never provided
goog.require('goog.array');
^^^^^^^^^^^^^^^^^^^^^^^^^^

src/os/xt/peer.js:6: ERROR - required "goog.async.Deferred" namespace never provided
goog.require('goog.async.Deferred');
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/os/xt/peer.js:7: ERROR - required "goog.async.Delay" namespace never provided
goog.require('goog.async.Delay');
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/os/xt/peer.js:8: ERROR - required "goog.events" namespace never provided
goog.require('goog.events');
^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/os/xt/peer.js:9: ERROR - required "goog.events.Event" namespace never provided
goog.require('goog.events.Event');
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/os/xt/peer.js:10: ERROR - required "goog.events.EventType" namespace never provided
goog.require('goog.events.EventType');
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/os/xt/peer.js:11: ERROR - required "goog.log" namespace never provided
goog.require('goog.log');
^^^^^^^^^^^^^^^^^^^^^^^^

src/os/xt/peer.js:12: ERROR - required "goog.log.Logger" namespace never provided
goog.require('goog.log.Logger');
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/os/xt/peer.js:13: ERROR - required "goog.string" namespace never provided
goog.require('goog.string');
^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/os/xt/peer.js:14: ERROR - required "ol.array" namespace never provided
goog.require('ol.array');
^^^^^^^^^^^^^^^^^^^^^^^^

20 error(s), 0 warning(s)

Done in 4.19s.
```

**Comment by wallw-bits** [comment]
That script is a little out of date and probably needs the paths containing `node_modules/` in `gcc-args` to be `../../node_modules/`.

**Comment by bradh** [comment]
@wallw-bits We'd need to check whether we're in a yarn workspace or standalone checkout to tell where the `node_modules` are to be found?

**Comment by wallw-bits** [comment]
Yes, that's correct.

**Comment by bradh** [comment]
Looks like @schmidtk has this fixed as part of #618
